lib/tasks/recipe_puppy_tasks.rake

```diff
@@ -0,0 +1,21 @@
+require File.expand_path('../../../config/environment', __FILE__)
+require 'utility/recipe_puppy_parser'
+
+namespace :recipe_parser do
+  desc "Scrape all recipes and ingredients from recipe puppy"
+  task :scrape_all do
+    Utility::RecipePuppyParser.new.save_all_recipes_to_database
+  end
+
+  desc "Query recipe from recipe puppy"
+  task :query_recipe do
+    Utility::RecipePuppyParser.new.query("Pasta Salad")
+  end
+end
+
+namespace :database do
```

Comment:
For something like this, which is semantically connected to both Recipe Puppy and IMSDB, we would want this to live in a separate file, probably `lib/tasks/database.rake`

However, this task also already exists. `bundle exec rails db:reset`